share/clus_apis.go

```diff
@@ -2492,26 +2508,43 @@ const (
 )
 
 const (
-	CLUSDlpDefaultSensor = "sensor.dlpdfltnv"
-	CLUSDlpSsnSensor     = "sensor.ssn"
-	CLUSDlpCcSensor      = "sensor.creditcard"
-	CLUSWafDefaultSensor = "sensor.wafdfltnv"
-	CLUSWafLog4shSensor  = "sensor.log4shell"
-	CLUSWafSpr4shSensor  = "sensor.spring4shell"
+	CLUSDlpDefaultSensor    = "sensor.dlpdfltnv"
+	CLUSFedDlpDefaultSensor = "fed.sensor.dlpdfltnv"
+	CLUSFedDlpDefSyncSensor = "fed.sensor.dlpdfltsyncnv"
+	CLUSDlpSsnSensor        = "sensor.ssn"
+	CLUSDlpCcSensor         = "sensor.creditcard"
+	CLUSDlpDefaultFedSensor = "fed.sensor.dlpdfltnv"
```

Comment:
CLUSDlpDefaultFedSensor & CLUSFedDlpDefaultSensor have the same value
Where is CLUSDlpDefaultFedSensor referenced?

Reply:
thanks for pointing out, I have removed redundant 'CLUSDlpDefaultFedSensor'